### ISSUE TITLE
Develop

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#138]
Implement "Play from here"

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2015-08-02T09:08:07Z

**Comment by kaaholst** [comment]
I think it should fix it.

EventBus [issue 149](https://github.com/greenrobot/EventBus/issues/149#issuecomment-76316437) references a [commit](https://github.com/greenrobot/EventBus/commit/be04a2de66981e819f40afd389e417e98c94c7da), which implement the recommended workaround by [yuzeh](https://github.com/yuzeh), who's fork we previously referenced in Squeezer.

I haven't been able to trigger the problem though, so I don't know if 2.4.1 actually fixes it.

I don't understand the supplied stacktrace though. When I open SubscriberMethodFinder.java from eventbus-2.4.1-sources.jar, line 76 is: `Method[] methods = clazz.getDeclaredMethods();`, and this line is protected by a try-catch (Throwable) and falls back to getMethods(). So how can it throw a NoSuchMethodException?


Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2016-05-15T21:12:34Z

**Comment by kaaholst** [comment]
I've got a nagging feeling this was caused by the Instant Run feature in newer versions of Android Studio (where it hot-patches new code in to the app instead of deploying a fresh APK). I turned that off, and can't reproduce.

1.4.1-beta contains 2.4.1 now (no crashes in Crashlytics that I can see), and the eventbus3 branch has https://github.com/nikclayton/android-squeezer/commit/15e08b029e8af97516152de71acc55cba6260d83 which is all the work (I think) necessary to use eventbus v3. I'll merge that in after 1.4.1 is released.


Original issue reported on github.com/nikclayton/android-squeezer by nikclayton on 2016-06-04T23:21:58Z
